### PR TITLE
fix(front-end): Fixed regressions and tweaked small things

### PR DESCRIPTION
**Pull request by nchapman**
Fixes are here!

@6a68 @johngruen @pdehaan r?

### CODE REVIEW

File: app/scripts/presenters/user_page_presenter.js [hunk 1]
```diff
@@ -40,7 +40,11 @@ define([
     },
 
     faviconUrl: function () {
-      return this.extractedFaviconUrl || '/images/icon-favicon_default@2x.png';
+      if (this.extractedFaviconUrl) {
+        return imageProxy.display(this.extractedFaviconUrl);
```
Comment (nchapman) [inline app/scripts/presenters/user_page_presenter.js]: Fixes mixed content errors.

File: app/scripts/templates/global_header/search_box.html [hunk 2]
```diff
@@ -1,3 +1,3 @@
 <form class='l-flex'>
-  <input class='query' type="text" placeholder="Search" />
+  <input class="query" type="text" placeholder="Search" />
```
Comment (nchapman) [inline app/scripts/templates/global_header/search_box.html]: Consistency ftw!

File: app/scripts/templates/user_pages/item.html [hunk 3]
```diff
@@ -2,11 +2,11 @@
   <img src="{{imageUrl}}" />
 </a>
 <div class="provider">
-  <img class="favicon" src="{{faviconUrl}}" />
+  <img class="favicon" src="{{faviconUrl}}" alt="favicon" />
```
Comment (nchapman) [inline app/scripts/templates/user_pages/item.html]: Accessibility ftw!

File: app/scripts/views/visits/index.js [hunk 4]
```diff
@@ -23,7 +23,7 @@ define([
       this.lastVisitId = null;
       this.hasMoreVisits = true;
       this.loadingMoreVisits = false;
-      this.count = 100;
+      this.count = 50;
```
Comment (nchapman) [inline app/scripts/views/visits/index.js]: Fewer images to load.

File: app/styles/_layout.scss [hunk 5]
```diff
@@ -4,7 +4,7 @@ body {
   font-family: $body-font;
 }
 
-.visits {
+#stage {
```
Comment (nchapman) [inline app/styles/_layout.scss]: Header is global so we always need some space above `#stage`.

**Comment by johngruen** [comment]
@nchapman LGTM

File: app/styles/modules/_user_page.scss [hunk 6]
```diff
@@ -1,4 +1,4 @@
-.visit {
+.user-page {
```
Comment (nchapman) [inline app/styles/modules/_user_page.scss]: `.user-page` is now the reusable concept.

File: app/scripts/views/visits/index.js (hunk 4 above)
Comment (6a68) [inline app/scripts/views/visits/index.js]: might be nice to add one of those "defer image loading until scrolled into view" things at some point

File: app/scripts/templates/global_header/search_box.html [hunk 7]
```diff
@@ -1,3 +1,3 @@
 <form class='l-flex'>
```
Comment (johngruen) [inline app/scripts/templates/global_header/search_box.html]: @nchapman: this should be double quoted as well!

Comment (nchapman) [inline app/scripts/templates/global_header/search_box.html]: On it!

File: app/scripts/views/visits/index.js (hunk 4 above)
Comment (nchapman) [inline app/scripts/views/visits/index.js]: Yeah or just have the images cached and ready to go.

**Pull request merged**